src/Offer/ReadModel/JSONLD/PropertyPolyfillOfferRepository.php

```diff
@@ -176,8 +176,9 @@ private function polyfillBrokenSameAs(array $json): array
 
         $urlParts = explode('/', $json['@id']);
         $id = array_pop($urlParts);
+        $name = $json['name']['nl'] ?? $json['name'];
```

Comment:
I think there can be events with translated names, but for example only `en`? Should we try use the main language to define the language index. Only when the main language fails we use the name if it is a string.

Reply:
I don't mind checking specifically for `nl` and using that if available, but if no `nl` name is found it will now use `$json['name']` which is not a string but an associative arrays with other keys like `fr`, `en`, ...

Reply:
I also seem to remember that the name actually does not matter very much to make the link work and I've seen links that just use `x` as slug (since it's the id in the URL that actually determines what gets shown and the slug is just for SEO). So if there's no `nl` maybe we can just use that?

Reply:
I've changed the code the prefer Dutch(since UiTinVlaanderen is in Dutch). And provide fallbacks for events without Dutch translation. I'll test with UiV will do with an event that doesn't have a Dutch translation.

Reply:
Still, the last fallback is `?? $json['name']` which is not a string. What you can do if there's no `nl` and for some weird reason the main language is also not set on `$json['name']` (which should not happen), is just grab the first value from the array. So something like `current($json['name'])`

Reply:
Correct, I was confused because in the past there were events without translations, but that was fixed with a previous replay.

Reply:
Ah yep that should be fixed by now indeed